models/issue_comment.go

```diff
@@ -104,7 +108,9 @@ type Comment struct {
 	UpdatedUnix int64     `xorm:"INDEX updated"`
 
 	// Reference issue in commit message
-	CommitSHA string `xorm:"VARCHAR(40)"`
+	CommitSHA    string        `xorm:"VARCHAR(40)"`
+	RepoFullName string        `xorm:"VARCHAR(200)"`
```

Comment:
This field seems unnecessary; if we need to know what the repo's name is, we can find that out by using `c.Issue.Repo`. In fact, we already do this in a lot of places (e.g. [Comment.HTMLURL()](https://github.com/go-gitea/gitea/blob/240609432b84a8caa5ec4c99a916277952ef02d5/models/issue_comment.go#L153))

Reply:
done.